src/NosCore.Controllers/Miniland/MinilandObjects/MgPacketHandler.cs

```diff
@@ -81,15 +81,15 @@ public override Task Execute(MinigamePacket minigamePacket, ClientSession client
             switch (minigamePacket.Type)
             {
                 case 1:
-                    Play(game);
+                    await Play(game);
                     break;
 
                 case 2:
-                    BroadcastEffect();
+                    await BroadcastEffect();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)